js/components/form/zone-completer.vue

```diff
@@ -120,31 +119,33 @@ export default {
                     opt.push('</div>');
                     return opt.join('');
```

Comment:
I agree, but I wanted to limit the diff. These components will be rewritten on the dev branch in pure Vue.js to drop selectize (and so drop jquery)